### ISSUE TITLE
choco install jq

### ISSUE THREAD

**Issue by pkoppstein** [stedolan/jq#920]
(1) Can choco be asked to point to jq 1.5?

(2) Currently, it fails anyway:

```
C:\WINDOWS\system32>choco install jq
Chocolatey v0.9.9.8
Installing the following packages:
jq
By installing you accept licenses for the packages.

jq v1.4.0.1
The package jq wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider setting
 'allowGlobalConfirmation'. Run 'choco feature -h' for more details.
Do you want to run the script?
 1) yes
 2) no
 3) print
1
 Attempt to get headers for http://stedolan.github.io/jq/download/win64/jq.exe failed.
   Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
 Downloading jq 64 bit
   from 'http://stedolan.github.io/jq/download/win64/jq.exe'
 WARNING: Write-ChocolateyFailure is deprecated. If you are the package maintainer, please use 'throw $_.Exception'
 instead.
 Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
 At C:\ProgramData\chocolatey\helpers\functions\Write-ChocolateyFailure.ps1:24 char:3
 +   throw "$failureMessage"
 +   ~~~~~~~~~~~~~~~~~~~~~~~
     + CategoryInfo          : OperationStopped: (Exception calli...04) Not Found.":String) [], RuntimeException
     + FullyQualifiedErrorId : Exception calling "GetResponse" with "0" argument(s): "The remote server returned an err 
    or: (404) Not Found."
The install of jq was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\jq\tools\chocolateyInstall.ps1'.
 See log for details.

Chocolatey installed 0/1 package(s). 1 package(s) failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures:
 - jq
```

**Comment by dtolnay** [comment]
The [Chocolatey website](https://chocolatey.org/packages/jq) links to [this repo](https://github.com/svnpenn/p) where I submitted a [pull request](https://github.com/svnpenn/p/pull/1/files) to fix jq 1.4 a few days before the release of 1.5. I will close it and make another pull request to go straight to 1.5. I don't know what the following submission/approval process looks like for Chocolatey, but hopefully @svnpenn can navigate that.

**Comment by dtolnay** [comment]
Pull request to go to 1.5 is [here](https://github.com/svnpenn/p/pull/2).

**Comment by svnpenn** [comment]
Hm, for some reason I never saw the first PR, I will look into it today, thanks

**Comment by svnpenn** [comment]
@dtolnay for some reason I never saw the first PR, I will look into it today, thanks

**Comment by svnpenn** [comment]
@dtolnay the package is now available here

http://chocolatey.org/packages/jq/1.5

Should be approved in about one day, then you should see it here

http://chocolatey.org/packages/jq

**Comment by nicowilliams** [comment]
It does require a workaround to build on Windows, you have to add `LDFLAGS=-lshwlapi` to the ./configure incantation.

**Comment by svnpenn** [comment]
@nicowilliams no, Chocolatey pulls the prebuilt EXEs only, it does not build from source.

**Comment by pkoppstein** [comment]
@svnpenn - Thanks for making 1.5 available.  Since http://chocolatey.org/packages/jq has a link to the new version, and since "choco install jq -version 1.5" did work, I'm closing this ticket, but please note there were two warnings:
```
WARNING: Write-ChocolateySuccess is deprecated. If you are the maintainer, please remove it from your package file.
...
Attempted to delete bad package install path if existing. Had an error:
 Access to the path 'jq.nupkg' is denied.
```

This was on a W10 machine.

**Comment by dtolnay** [comment]
@pkoppstein I have filed that as https://github.com/svnpenn/p/issues/3.

**Comment by worldsayshi** [comment]
Is there any workaround for installing 1.4.* in the meantime?

**Comment by svnpenn** [comment]
@worldsayshi I can fix 1.4, but why do you need it?

**Comment by worldsayshi** [comment]
Not really. Since 1.5 wasn't the one being pointed to I assumed 1.5 was still considered unstable or something. I ended up using 1.5 however and it seems to work well. :)

**Comment by worldsayshi** [comment]
@svnpenn Not really. Since 1.5 wasn't the one being pointed to I assumed 1.5 was still considered unstable or something. I ended up using 1.5 however and it seems to work well. :)

**Comment by dtolnay** [comment]
If it were up to us, it would have pointed to 1.5 a long time ago. We are just waiting on the Chocolatey moderation process.

**Comment by svnpenn** [comment]
@dtolnay and all, 1.5 has been approved now, enjoy

http://chocolatey.org/packages/jq

**Comment by dtolnay** [comment]
Great, I will bump the version number on our download page tonight. Thanks for your help!

**Comment by cup** [comment]
https://github.com/stedolan/jq/issues/1975
